### DISCUSSION TITLE
Smartcard plugin

### DISCUSSION THREAD

**Discussion by Canosus** [FreeRDP/FreeRDP#6774] [Q&A] [answered]
Good day, i have a question about `--plugin rdpdr --data smartcard:<name> --`. I have several types of cards and several cards as well, so if i put * as card name will it work or i need to list all the cards?

**Comment by akallabeth** [comment, marked as answer]
1. Please do not use `freerdp 1` style command line arguments (the warning about 'deprecated' is there for a reason)
2. `/smartcard` by default redirects all smartcards detected by `pcsc`
